openapi/v1/core-data.yaml

```diff
@@ -954,6 +954,12 @@ components:
           type: array
           items:
             $ref: '#/components/schemas/reading'
+        tags:
+          title: tags
+          uniqueItems: false
+          type: array
```

Comment:
the type could a pure object without any reference, but the example could be identified clearly.
please see the example of protocol properties
https://github.com/edgexfoundry/edgex-go/blob/9f0651306d868a93c4f6d04f573f0b50f68ed542/openapi/v1/core-metadata.yaml#L2210-L2214